torchvision/prototype/transforms/_geometry.py

```diff
@@ -300,12 +295,11 @@ def __init__(
         self.center = center
 
     def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
-        angle = float(torch.empty(1).uniform_(float(self.degrees[0]), float(self.degrees[1])).item())
+        angle = torch.empty(1).uniform_(self.degrees[0], self.degrees[1]).item()
```

Comment:
Remove unnecessary float convertions. The output is guaranteed to be float. Same for all occurrences below.